### PR TITLE
[GECO-64] show version number in footer

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

File: cassiopeia/pom.xml [hunk 1]
```diff
@@ -20,7 +20,8 @@
         <tomcat.server.id>TomcatServer</tomcat.server.id>
 
         <admin.password>admin</admin.password>
-
+        <pullrequest></pullrequest>
+        <buildNumber></buildNumber>
```
Comment (jdamerow) [inline cassiopeia/pom.xml:24]: this also needs the buildnumber plugin which I don't believe is included right now.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by snilapwa** [comment]
updated with scm definition

**Comment by jdamerow** [comment]
Please add buildNumber.properties to gitignore

**Comment by snilapwa** [comment]
Updated gitignore

**Review by jdamerow — APPROVED** [review]

**Pull request merged**